backend/app.py

```diff
@@ -6,62 +6,63 @@
 - jq python api docs: https://github.com/mwilliamson/jq.py
 """
 import json
-import os.path  # TODO: Siggie confused: are you supposed to use path.<method>(...) or os.path.<method>(...)?
+import os
 import errno
 from pathlib import Path
-from subprocess import PIPE, Popen, call as sp_call
+from subprocess import call as sp_call
 from typing import Any, Dict, List, Union
 
 import numpy as np
 import pandas as pd
-
 import requests
 import uvicorn
 from fastapi import FastAPI, Query
 from fastapi.middleware.cors import CORSMiddleware
+from pandasql import sqldf
 from pydantic import BaseModel
 
 from enclave_wrangler.config import config, FAVORITE_DATASETS
 
 import jq
 
-from pandasql import sqldf
-pysqldf = lambda q: sqldf(q, globals())
 
 DEBUG = True
 PROJECT_DIR = Path(os.path.dirname(__file__)).parent
 OBJECTS_PATH = f'{PROJECT_DIR}/termhub-csets/objects'
 CSETS_JSON_PATH = f'{OBJECTS_PATH}/OMOPConceptSet/latest.json'
 CONCEPTS_JSON_PATH = f'{OBJECTS_PATH}/OMOPConcept/latest.json'
 CONCEPT_SET_VERSION_ITEM_JSON_PATH = f'{OBJECTS_PATH}/OmopConceptSetVersionItem/latest.json'
-
 CSV_PATH = f'{PROJECT_DIR}/termhub-csets/datasets'
 
 # load big files!
 # TODO: this is too slow for development where the backend has to restart all the time
-#       @Joe: would it be too crazy to run two backend servers, one to hold the data
-#             and one to service requests and handle logic? probably....
-ds = { name: pd.read_csv(os.path.join(CSV_PATH, name + '.csv')) for name in FAVORITE_DATASETS}
-print(ds.keys())
-
-concept = ds['concept']
-cnts = pysqldf("""
+#  @Joe: would it be too crazy to run 2 backend servers, 1 to hold the data and 1 to service requests / logic? probably.
```

Comment:
@Sigfried 
```
#  @Joe: would it be too crazy to run 2 backend servers, 1 to hold the data and 1 to service requests / logic? probably.
#  @Siggie: Not a bad idea. If we invest time in that, may be better to do RDBMS instead, but I think running 2
#   *might* not take too much time... actually maybe it would. Need to pass data between the processes. RCP? We could
#   do it over REST, but idk. Maybe worth looking into / trying for an hour.
```